recipes/pytest-codeblocks/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-codeblocks" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f95ab225401a4a55d9992c1dff3c5a7c80042c83738db8069d1a601f9a4c0e9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
```

Comment:
Shouldn't this have `dataclasses` and `importlib_metadata`?

Reply:
Fixed now. (importlib_metadata wasn't necessary, removed from upstream.)